git-hub

```diff
@@ -87,15 +103,292 @@ def push():
+        repo_root = pjoin(path_repo, '..')
+    path_git = pjoin(path_repo, ".git")
+    return path_git
+
+
+def find_in_dictionary(pr_num, open_or_closed_dictionary):
+    """Fetches all the information on a certain PR
+
+    Parameters
+    ----------
+    pr_num : integer
+        number of PR to fetch information from
+    open_or_closed_dictionary : dictionary
+        dictionary of open or closed PRs to find number in
+    """
+    if(open_or_closed_dictionary is None):
+        return None, None
+    for dictionary in open_or_closed_dictionary:
+        for sub_keys in list(dictionary.keys()):
+            if(sub_keys == str(pr_num)):
+                return sub_keys, dictionary[sub_keys]
+    return None, None
+
+
+def get_info(pr_num):
+    """Fetches all the information on a certain PR
+
+    Parameters
+    ----------
+    pr_num : integer
+        number of PR to fetch information from
+    """
+    path_git = find_path_to_git()
+    path_github = pjoin(path_git, 'git-hub')
+    path_prs = pjoin(path_github, 'pull-requests.toml')
+    f = open(path_prs, "r")
+    pr_dict = toml.load(f)      # fetches toml file and creates a dictionary
+    open_dict = pr_dict['open pull requests']
+    closed_dict = pr_dict['closed pull requests']
+
+    try:
+        open_PR = "O"
+        key, pr_dictionary = find_in_dictionary(pr_num, open_dict)
+        if(pr_dictionary is None):
+            open_PR = "C"
+            key, pr_dictionary = find_in_dictionary(pr_num, closed_dict)
+        if(pr_dictionary is None):
+            click.echo("Could not find PR #{}. Run 'git hub sync' and try again.".format(pr_num))
+        else:
+            reviewers = "None"
+            if(pr_dictionary['reviewers'] is not ""):
+                reviewers = pr_dictionary['reviewers']
+            assignee = "None"
+            if(pr_dictionary['assignee'] is not ""):
+                assignee = pr_dictionary['assignee']
+            milestone = "None"
+            if(pr_dictionary['milestone'] is not ""):
+                milestone = pr_dictionary['milestone']
+            labels = "None"
+            if(pr_dictionary['labels'] is not ""):
+                labels = pr_dictionary['labels']
+            click.echo('{0} \x1b[0;32;40m {1} \x1b[0m {2}/{3} {4}'.format(key, open_PR, pr_dictionary['user'], pr_dictionary['branch'], pr_dictionary['comment']))
+            click.echo('-Labels: {0}  -Reviewers: {1}  -Assignees: {2}  -Milestones: {3}'.format(labels, reviewers, assignee, milestone))
+    except (OSError, IOError) as e:
+        # if pull-requests.toml hasnt been created yet calls sync and then reties to fetch
+        sync()
+        get_info(pr_num)
+
+
+def print_in_order(sort, q):
+    """
+    Prints items in q sorted by order given by sort
+
+    Parameters
+    ----------
+    sort : string
+        sort in increasing or decreasing order
+    q : PriorityQueue
```

Comment:
Why not just sort a list?  Sort can take a function argument that determines which keys are used for sorting.